front/src/common/SpeedLimitByTagSelector/SpeedLimitByTagSelector.tsx

```diff
@@ -1,11 +1,12 @@
-import { useMemo } from 'react';
+import { useEffect, useMemo } from 'react';
 
 import { Select } from '@osrd-project/ui-core';
 import cx from 'classnames';
 import { isEmpty } from 'lodash';
 import { useTranslation } from 'react-i18next';
 
 import { createStringSelectOptions } from 'utils/uiCoreHelpers';
+import { DEFAULT_COMPOSITION_CODE } from 'applications/stdcm/consts';
```

Comment:
Got a little eslint warning : 
`applications/stdcm/consts` import should occur before import of `utils/uiCoreHelpers`

Reply:
I removed the import so we don't have this warning anymore